### PR TITLE
[tests-only] [full-ci] Add spaces tests on `apiWebdavProperties2` suites

### LINKED ISSUE #3035: add spaces webdav for `apiWebdavProperties2` suite
### Description
- add spaces WebDAV URL for `apiWebdavProperties2` suite from `oc/core`
- includes:
  - getFileProperties.feature

### Related
- part of https://github.com/owncloud/ocis/issues/2854

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
adds spaces DAV path in `apiAuthWebDav` suite

## Related Issue

- fixes https://github.com/owncloud/ocis/issues/3035

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by SwikritiT — COMMENTED** [review]
LGTM

File: tests/acceptance/features/bootstrap/WebDav.php [hunk 1]
```diff
@@ -1885,7 +1892,7 @@ public function listFolder(
 			$this->getStepLineRef(),
 			$properties,
 			$type,
-			($this->usingOldDavPath) ? 1 : 2
+			$davPathVersion
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/WebDav.php:1895]:
```suggestion
			$this->getDavPathVersion()
```
IMO it's good to just provide the DAV path here rather introducing the function param.

File: tests/acceptance/features/bootstrap/WebDav.php [hunk 2]
```diff
@@ -1860,6 +1860,7 @@ public function asExactlyOneOfTheseFilesOrFoldersShouldExist(string $user, strin
 	 * @param string $folderDepth requires 1 to see elements without children
 	 * @param array|null $properties
 	 * @param string $type
+	 * @param string|null $davPathVersion
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/WebDav.php:1863]: this param is not used anymore..can be removed!

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by phil-davis — APPROVED** [review]

**Comment by amrita-shrestha** [comment]
This PR change makes apiFavorites/favorites.feature and apiShareOperationsToShares2/getWebDAVSharePermissions.feature tests fail on ocis. So, I am currently working on that.

**Pull request merged**